model/rpc/spans.yaml

```diff
@@ -56,11 +75,17 @@ groups:
         requirement_level: recommended
       - ref: network.type
         requirement_level: recommended
+    events: [rpc.message]
 
   - id: rpc.grpc
     type: span
+    stability: experimental
     extends: rpc
-    brief: 'Tech-specific attributes for gRPC.'
+    brief: >
+      Semantic Conventions for [gRPC](https://grpc.io/) extend and override the
+      [RPC client and server spans](rpc-spans.md)
```

Comment:
this is one of the several reasons we can't ~~have  nice things~~ generate span kind in MD - we define spans without kind, but assume it can be client or server.

TODO: create issue to require kind and clean up stuff.